CMakeLists.txt

```diff
@@ -46,6 +46,14 @@ target_link_libraries(libvroom PUBLIC OpenSSL::SSL)
 target_link_libraries(libvroom PUBLIC Threads::Threads)
 
 if(WIN32) 
+
+  #disable iterator base class deprecation warning
+  add_compile_definitions(_SILENCE_CXX17_ITERATOR_BASE_CLASS_DEPRECATION_WARNING)
+
+  #additional library dependency  
+  target_link_libraries(vroom PRIVATE Crypt32.lib)
+  target_link_libraries(libvroom PRIVATE Crypt32.lib)  
```

Comment:
This is just about getting the static library instead of the dll right? I don't really know if this is a good way of specifying static libs instead of dynamic, I'm not really familiar enough with the whole building tool chain.. But it seems to me that every one who wants to build this on Win now will need the statib lib instead of the dynamic lib?